acos_client/v30/slb/hm.py

```diff
@@ -23,16 +23,25 @@
 class HealthMonitor(base.BaseV30):
 
     # Valid method objects
+    UDP = 'udp'
     ICMP = 'icmp'
     TCP = 'tcp'
     HTTP = 'http'
     HTTPS = 'https'
     url_prefix = "/health/monitor/"
+    UDP_CONNECT = 'udp'
 
     _method_objects = {
         ICMP: {
             "icmp": 1
         },
+        UDP: {
+            "udp": 1,
+            "udp-port": 5555,
+            "force-up-with-single-healthcheck": 0
+        },
+        UDP_CONNECT: {
```

Comment:
Why does this have an empty payload?

Reply:
Actually why not just use the UDP entry instead?

Reply:
We were facing weird issue when sharing udp and udp connect dict with axapi.
when we passed port and udp=1, there was error (JSON field is disabled by another field)

So, created another story to investigate this (as resolution of this will also lead to changes in a10-octavia):
https://a10networks.atlassian.net/browse/STACK-647

for now, we can get it merged as it works.